src/conditions/base/evm.ts

```diff
@@ -10,6 +10,7 @@ export const STANDARD_CONTRACT_TYPES = ['ERC20', 'ERC721'];
 export class EvmCondition extends Condition {
   public readonly schema = Joi.object({
     contractAddress: Joi.string().pattern(ETH_ADDRESS_REGEXP).required(),
+    // TODO why is chain a string here?
     chain: Joi.string()
```

Comment:
Probably a mistake, `number` works just fine
```suggestion
    chain: Joi.number()
```

Reply:
I figured, but wanted to be sure. Fixed now.